ci/toolchain_manager.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+# Builds and publishes kubesaw components to a quay repository.  Used as a part
+# of the CI process for testing in openshift-ci.
+
+set -e
+
+# parse input
+BRANCH=${BRANCH:-pubviewer-mvp}
+QUAY_NAMESPACE=${QUAY_NAMESPACE:-konflux-workspaces}
+TAG=${2:-e2etest}
+KUBECLI=${KUBECLI:-kubectl}
+
+HOST_OPERATOR_REPO=${HOST_OPERATOR_REPO:-https://github.com/filariow/host-operator.git}
```

Comment:
I assume this is also temporary, why not create a downstream repo for each of these projects under workspace?

Reply:
Lets add a story capturing that